packages/ui-extensions/src/surfaces/admin/components/Switch/examples/settings-panel-with-stack.jsx

```diff
@@ -0,0 +1,9 @@
+<s-stack gap="base">
```

Comment:
```suggestion
<s-stack gap="small-200">
```

Maybe just personal preference, but the example feels spacey. Perhaps using `small-200` might group the settings panel better 🎛️